packages/best-practices/test/rules/casing.rule.test.ts

```diff
@@ -0,0 +1,22 @@
+import { RuleTester, createRuleTester, createTestRunner } from "@typespec/compiler/testing";
+import { casingRule } from "../../src/rules/casing.rule.js";
+
+describe("casing rule", () => {
+  let ruleTester: RuleTester;
+
+  beforeEach(async () => {
+    const runner = await createTestRunner();
+    ruleTester = createRuleTester(runner, casingRule, "@typespec/my-linter");
+  });
+
+  it("emit diagnostics when using model named foo", () => {
+    ruleTester.expect(`model Foo {}`).toEmitDiagnostics({
+      code: "@typespec/my-linter:no-foo-model",
```

Comment:
There is this change that  in the linter library the resulting diagnostic code was `@typespec/my-linter/no-foo-model` but will now be `@typespec/my-linter:no-foo-model` ( `/` vs `:` to join the library name and code name)

Using `:` is very useful to be able to split the library name from the rule name which is needed to be able to load the library.

This itself is not a breaking change but migrating from the linter library to this will create this breaking change(if the user enables the new linter rules)

Reply:
or we don't have `:` as the separator and just use `/` and maybe instead make sure the rule name doesn't contain a `/`

Reply:
Discussed offline and estabilish that using going back to using `/` would be the right move.